resource/helpers/recipes-helper.py

```diff
@@ -104,13 +104,17 @@ def add_gpu_options_to_resp(response):
             'value': 'custom'
         }
     ]
-    response["gpu_memory_choices"] = [
+    response["gpu_memory_allocation_mode"] = [
         {
             'label': 'No limitation',
             'value': 'all'
```

Comment:
+1 for doing an enum for possible values

Reply:
Yop, will do in a future PR